integration_tests/src/test/kotlin/utils/CommandHelper.kt

```diff
@@ -4,10 +4,24 @@ import java.io.File
 import java.util.concurrent.TimeUnit
 
 fun String.runCommand(workingDir: File): ProcessResult = ProcessBuilder(*split("\\s".toRegex()).toTypedArray())
-    .directory(workingDir)
-    .redirectOutput(ProcessBuilder.Redirect.PIPE)
-    .redirectError(ProcessBuilder.Redirect.PIPE)
-    .start().run {
-        waitFor(60, TimeUnit.MINUTES)
-        ProcessResult(exitValue(), inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText())
-    }
+        .directory(workingDir).run {
+            if (System.getProperty("os.name").startsWith("Windows"))
+                redirectOutput(File(outputFileName)).
+                redirectError(File(errorFileName)).
+                start().
+                    run {
+                        waitFor(processTimeout, TimeUnit.MINUTES)
+                        ProcessResult(exitValue(), File(outputFileName).readText() + File(errorFileName).readText())
+                    }
+            else redirectOutput(ProcessBuilder.Redirect.PIPE).
+                redirectError(ProcessBuilder.Redirect.PIPE).
+                start().
+                    run {
+                        waitFor(processTimeout, TimeUnit.MINUTES)
+                        ProcessResult(exitValue(), inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText())
+                    }
+        }
```

Comment:
I am pretty sure that you Guys could wrap it to nice method instead of nested if, run etc

Reply:
Yes, this file needs some polish 😄

Reply:
done